mempool/reactor_test.go

```diff
@@ -64,6 +65,66 @@ func TestReactorBroadcastTxsMessage(t *testing.T) {
 	waitForTxsOnReactors(t, txs, reactors)
 }
 
+// regression test for https://github.com/tendermint/tendermint/issues/5408
+func TestReactorConcurrency(t *testing.T) {
+	config := cfg.TestConfig()
+	const N = 2
```

Comment:
What does this actually do? Seems like we're only testing two reactors regardless of this setting, in which case I think we can just drop this constant.

Reply:
we use it when creating reactors makeAndConnectReactors. the number can be increased to 4 for example. 2 is only required when we want to assert the order of transactions on the receiving side

Reply:
Sure, but what purpose would it serve to start more reactors than the two we're testing?

Reply:
I can't think of any